aya/src/sys/bpf.rs

```diff
@@ -406,6 +407,54 @@ pub(crate) fn bpf_link_create(
     unsafe { fd_sys_bpf(bpf_cmd::BPF_LINK_CREATE, &mut attr) }
 }
 
+// since kernel 5.7
+pub(crate) fn bpf_tcxlink_create(
```

Comment:
nit: `bpf_tcxlink_create` is not a syscall.
You're doing `bpf_link_create` here and we already have a helper function for that.
The only difference in your case is that you're passing `ordering: LinkOrder`

You could do this by changing the signature of `bpf_link_create` to take `Option<LinkOrder>`... then you'd avoid a lot of duplicated code.

Reply:
Done